src/account-info/AccountDetailsScreen.js

```diff
@@ -29,29 +36,51 @@ class AccountDetailsScreen extends PureComponent<Props> {
   };
 
   render() {
-    const { user } = this.props;
+    const { user, isDeactivated } = this.props;
     const title = {
       text: '{_}',
       values: {
         // This causes the name not to get translated.
         _: user.full_name || ' ',
       },
     };
-
-    return (
-      <Screen title={title}>
-        <AccountDetails user={user} />
+    let deactivatedText = null;
+    let privateMessageButton = null;
+    if (isDeactivated) {
+      deactivatedText = (
+        <RawLabel style={styles.deactivatedText} text="(This user has been deactivated)" />
```

Comment:
Can we use `<Label />` instead so that this text gets translated in user selected language.

Reply:
I had used label first. It does not adapt to theme changes ( dark mode ).

Reply:
Can you please provide more details on this? maybe screenshot.

Then if there are some layout issues, then it will be better to first enhance `Label` to make it look better in dark theme.

Reply:
Is there a way that text can be translated, before it is added to the RawLabel?

Reply:
**Fixed**